app/Integration/Academic/Mapper/FromAcademic/FromAcademicMapper.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Integration\Academic\Mapper\FromAcademic;
+
+use Acme\Academic\Academic;
+
+interface FromAcademicMapper
```

Comment:
Not sure FromAcademic it's a good name. De facto it's a serializer, and being an action, it's better use a verbs instead of FromSomething. So what about call it AcademicSerializer (and AcademicHydrator the other class)